abis_mapping/templates/survey_metadata/mapping.py

```diff
@@ -40,28 +41,31 @@ def apply_validation(
         Returns:
             frictionless.Report: Validation report for the specified data.
         """
-        # Construct Resource (Table with Schema)
+        # Construct Schema
+        schema = frictionless.Schema.from_descriptor(self.schema())
+
+        # Construct TableResource
         resource = frictionless.Resource(
```

Comment:
Silly question: why this is not a `TableResource` here like the others?

Reply:
No not silly at all. This still works so i think maybe it was another change that may have fixed a previous issue i was having (i think it had something either to do with the keyword arg being `data=data` or using the resource open method). I will see if i can change the use of `TableResource` back to `Resource` and push. Leave it with me.

Reply:
OK i changed the use of `TableResource` back to plain old `Resource` which still works as intended. The comments within the `apply_mapping` and `apply_validation` methods suggests that a more generic resource is intended and use of `Resource` is more appropriate allowing for easier expansion beyond tabular data.